web/src/cypress/support/util.ts

```diff
@@ -162,3 +162,16 @@ export function testScreen(
     describe(screenName(), () => fn(screen()))
   })
 }
+
+export function setFavorites(ids: Array<string>): Promise<unknown> {
```

Comment:
Since this is user-specific it also shouldn't be in `util` but in the `users` support file with a corresponding name like `setFavoriteUser`.

Lastly, just have it handle a single user, return Cypress.Chainable and call it from `.filter(...).forEach( setFavoriteUser )`

Cypress will already handle chaining the events and does not actually support/use promises properly. Also treat it as a private helper and don't export it since we're really not adding support outside of `createManyUsers`

Reply:
addressed.